src/material-experimental/mdc-tabs/_tabs-common.scss

```diff
@@ -19,7 +19,7 @@ $mat-tab-animation-duration: 500ms !default;
     // is an opt-in behavior. Also technically we don't need to combine the two classes, but
     // we need the extra specificity to avoid issues with CSS insertion order.
     flex-grow: 0;
-    min-width: 160px;
+    min-width: 90px;
```

Comment:
If the `min-width` is in the spec, it seems like either MDC should set it directly. Could you check with the MDC folks about whether the omission of this from their styles is intentional?

I know this is really nitpicky for one style, but the goal of us basing our components on top of MDC is to converge as much as possible. Every extra line of style we add on top is a step away from that goal.

Reply:
MDC now exposes `min-width` (see [commit](https://github.com/material-components/material-components-web/commit/c4ab987221d5a3b9ab588321bb0347f5d665505a)). Once we bump our canary version (see https://github.com/angular/components/pull/21501) we can just delete our custom `min-width` and use the default one.